addons/mail/static/src/core/common/thread.js

```diff
@@ -393,7 +404,10 @@ export class Thread extends Component {
             };
         });
         useEffect(applyScroll);
-        useChildSubEnv({ onImageLoaded: applyScroll });
+        useChildSubEnv({
+            onImageLoaded: applyScroll,
+            onClickNotification: this.onClickNotification.bind(this),
```

Comment:
Ok for `onClickNotification` there for stable fix, in `master` should move `NotificationMessage` template and this function in `Message` component, as it's only used by this latter component.